code/core/src/types/modules/core-common.ts

```diff
@@ -453,6 +453,13 @@ export interface StorybookConfigRaw {
      */
     actions?: boolean;
 
+    /**
+     * Enable the onboarding checklist sidebar widget
+     *
+     * @default true
+     */
+    checklistWidget?: boolean;
```

Comment:
```suggestion
    onboardingInSidebar?: boolean;
```

I propose we link the featureFlag specifically to the sidebar, since that's what we're disabling.

I personally think the word "widget" has no meaning in storybook at the moment, so I'd prefer we avoided the word.

I agree with @Sidnioulz that `checklist` is a technical term, for the feature, and `onboarding` is more user-facing.